<?xml version="1.0"?>
<launch>
  <!--include file="$(find iiwa_moveit)/launch/moveit_planning_execution.launch">
    <arg name="sim" value="true"/>
  </include-->
  <node name="pickup" pkg="iiwa_myo" type="pickup" ns="iiwa" respawn="false" output="screen"/>

  <arg name="port" default="/dev/ttyACM0"/>
  <node name="myo_raw" pkg="ros_myo" type="myo-rawNode.py" args="$(arg port)" />

  <arg name="griphardt" default="/dev/ttyUSB0"/>
  <node name="robotiq_c_model_control" pkg="robotiq" type="CModelRtuNode.py" args="$(arg griphardt)" />

</launch>
